app/objects/player.py

```diff
@@ -488,12 +488,11 @@ async def restrict(self, admin: "Player", reason: str) -> None:
         """Restrict `self` for `reason`, and log to sql."""
         await self.remove_privs(Privileges.NORMAL)
 
-        log_msg = f'{admin} restricted for "{reason}".'
         await app.state.services.database.execute(
             "INSERT INTO logs "
-            "(`from`, `to`, `msg`, `time`) "
-            "VALUES (:from, :to, :msg, NOW())",
-            {"from": admin.id, "to": self.id, "msg": log_msg},
+            "(`from`, `to`, `action`, `msg`, `time`) "
+            "VALUES (:from, :to, 'restrict', :msg, NOW())",
+            {"from": admin.id, "to": self.id, "msg": reason},
```

Comment:
same here with 'restrict'

Reply:
solved in https://github.com/cmyui/gulag/commit/6710da3c927f7afdf628e4f7d74d97314d27adc3